### PR TITLE
add conductr-lib

### PR DESCRIPTION
**Pull request by SethTisue**
fyi @huntc

I can't actually test this quite yet because of another failure elsewhere in the build. I'll come back to it

### CODE REVIEW

**Comment by SethTisue** [comment]
~~ok, now we can test: https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1068/consoleFull~~

**Comment by SethTisue** [comment]
oops, that was testing the wrong thing, try again: https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1071/consoleFull (404 til Jenkins gets his woofers re-coned)

**Comment by SethTisue** [comment]
akka24ConductRClientLib needs akka-sse. new run: https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1073/consoleFull (404 til Jenkins steals all the lemons from the neighbors' backyard)

**Comment by SethTisue** [comment]
@huntc at https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1074/consoleFull I'm seeing compile errors like:

```
[conductr-lib] [error] /Users/tisue/community.212/target-0.9.7-RC1/project-builds/conductr-lib-665de1fd0541e59f8aa87181f02f7642f7fe3a22/akka24-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/akka/BundlesConnector.scala:21: object pattern is not a member of package de.heikoseeberger.akkasse
[conductr-lib] [error] import de.heikoseeberger.akkasse.pattern.Streams
[conductr-lib] [error]                                  ^
```

I believe it's because you're using an old version of akka-sse.

in your build I see this comment:

```
// Cannot use the client lib with 2.12 until Play also moves there, and we use an akka-sse
// for 2.12.
```

what way forward here do you suggest? can you upgrade and use the latest akka-sse? or should we not even try to build akka24ConductRClientLib here, for now?

**Comment by huntc** [comment]
That particular lib shouldn't be processed. We are using sbt-doge to provide +compile.

**Comment by SethTisue** [comment]
ok, trying again without akka24ConductRClientLib : https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1075/consoleFull

(I don't follow what you're saying about sbt-doge, but maybe I don't need to?)

**Comment by huntc** [comment]
Thanks Seth. In essence only each sub project that has a cross build setting for 2.12 needs including. There are quite a few that are required to be excluded.

**Comment by SethTisue** [comment]
here's where we stand on what subprojects we are currently attempting to build:

`[conductr-lib] [info] These subprojects will be built: root, play25ConductRBundleLib, play25ConductRClientLib, javaConductRBundleLib, javaTestLib, akka24ConductRBundleLib, scalaConductRClientLib, play25Common, scalaConductRBundleLib, akka24Common, scalaCommon, conductRBundleLib, javaCommon, common, akka24TestLib`

do you see anything there you know right away I should throw out? (The community build has Akka 2.4 and Play 2.6, so maybe for starters I should throw out the play25 things? I think 2.6 hasn't diverged that much yet, so not sure...)

**Comment by SethTisue** [comment]
trying it without the Play ones (`play25Common` failed to compile in a local test): https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1076/consoleFull

**Comment by SethTisue** [comment]
sweet, sweet green!

**Comment by huntc** [comment]
Looks good. Thanks again Seth. 

We cannot upgrade Akka-SSE right now (breaking changes). Thanks for working around all that.

File: community.dbuild [hunk 1]
```diff
@@ -820,6 +820,8 @@ build += {
     ]
   }
 
+  // forked (December 2016) to get rid of the usual bintray-sbt stuff that
+  // makes dbuild upset
```
Comment (cunei) [inline community.dbuild:836]: @SethTisue Please check: https://github.com/typesafehub/dbuild/commit/7ef6be74b8025033d61bb14011e52a27c9b44723
Bintray-sbt based projects should be ok with 0.9.7-RC1. In theory, at least.

Comment (SethTisue) [reply community.dbuild:836]: @cunei not in my experience — see https://github.com/typesafehub/dbuild/issues/158, which I would suggest reopening as the problem doesn't seem to be solved. (I don't have the necessary access to reopen it myself.)

**Pull request merged**